race condtion while running ocdev

Se this error intermittently while running ocdev operations:

```
# ocdev application create app2
Creating application: app2
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7aafb5]

goroutine 5 [running]:
github.com/redhat-developer/ocdev/pkg/notify.getLatestReleaseTag(0x0, 0x0, 0x0, 0x0)
        /home/gopath/src/github.com/redhat-developer/ocdev/pkg/notify/notify.go:32 +0x125
github.com/redhat-developer/ocdev/pkg/notify.CheckLatestReleaseTag(0x8ab394, 0x6, 0x0, 0x0, 0x0, 0x0)
        /home/gopath/src/github.com/redhat-developer/ocdev/pkg/notify/notify.go:46 +0xe2
github.com/redhat-developer/ocdev/cmd.getLatestReleaseInfo(0xc42001c420)
        /home/gopath/src/github.com/redhat-developer/ocdev/cmd/root.go:77 +0x4b
created by github.com/redhat-developer/ocdev/cmd.Execute
        /home/gopath/src/github.com/redhat-developer/ocdev/cmd/root.go:52 +0x6f

```

--- a/pkg/notify/notify.go
+++ b/pkg/notify/notify.go
@@ -20,19 +20,16 @@ const (
 
 // getLatestReleaseTag polls ocdev's upstream GitHub repository to get the
 // tag of the latest release
-func getLatestReleaseTag() (tag string, err error) {
+func getLatestReleaseTag() (string, error) {
 	client := github.NewClient(nil)
 	release, response, err := client.Repositories.GetLatestRelease(context.Background(), ghorg, ghrepo)
 	if response != nil {
-		defer func() {
-			if err = response.Body.Close(); err != nil {
-				err = errors.Wrap(err, "closing response body failed")
-			}
-		}()
-		tag = *release.TagName
-		return
+		defer response.Body.Close()
 	}
-	return
+	if err != nil {
+		return "", errors.Wrap(err, "error getting latest release")
+	}
+	return *release.TagName, nil
 }
 
 // CheckLatestReleaseTag returns the latest release tag if a newer latest

